website/docs/API.md

```diff
@@ -515,41 +515,43 @@ The `props` passed into the callback will be the `initialProps` provided in the
 
 ### `options` (Optional)
 
-A `RenderHookOptions` object to modify the execution of the `callback` function, containing the following properties:
+A `RenderHookOptions<Props>` object to modify the execution of the `callback` function, containing the following properties:
 
 #### `initialProps`
 
-The initial values to pass as `props` to the `callback` function of `renderHook`.
+The initial values to pass as `props` to the `callback` function of `renderHook`.  The `Props` type is determined by the type passed to or inferred by the `renderHook` call.
 
 #### `wrapper`
 
-A React component to wrap the test component in when rendering. This is usually used to add context providers from `React.createContext` for the hook to access with `useContext`. `initialProps` and props subsequently set by `rerender` will be provided to the wrapper.
+A React component to wrap the test component in when rendering. This is usually used to add context providers from `React.createContext` for the hook to access with `useContext`.
```

Comment:
While fixing the types, I also noticed that this is not true for your implementation either.